### ISSUE TITLE
ImportError: cannot import name 'appengine' from 'urllib3.contrib'

### ISSUE THREAD

**Issue by hyxxsfwy** [FederatedAI/FATE#5202]
**Describe the bug**
requests-toolbelt version conflict.

**To Reproduce**
1. Create venv `fate`: `conda create -n fate python=3.8`, then `conda activate fate`
2. Install fate_client: `pip install fate_client`
3. Run `flow -h`
4. See error `ImportError: cannot import name 'appengine' from 'urllib3.contrib'`

**Expected behavior**
Flow show the help message.

**Desktop (please complete the following information):**
 - OS: [Ubuntu 20.04 LTS]
 - Version [fate-client 1.11.3]

**Additional context**
```
Traceback (most recent call last):
  File "/home/xxx/miniconda3/envs/fate/lib/python3.8/site-packages/requests_toolbelt/_compat.py", line 48, in <module>
    from requests.packages.urllib3.contrib import appengine as gaecontrib
ImportError: cannot import name 'appengine' from 'requests.packages.urllib3.contrib'
```

`pip install -U 'requests-toolbelt>=1'` may solve the problem, but ERROR occurs while installing:

```
Installing collected packages: requests-toolbelt
  Attempting uninstall: requests-toolbelt
    Found existing installation: requests-toolbelt 0.9.1
    Uninstalling requests-toolbelt-0.9.1:
      Successfully uninstalled requests-toolbelt-0.9.1
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
fate-client 1.11.3 requires requests_toolbelt<0.10.0,>=0.9.1, but you have requests-toolbelt 1.0.0 which is incompatible.
fate-test 1.11.3 requires requests_toolbelt<0.10.0,>=0.9.1, but you have requests-toolbelt 1.0.0 which is incompatible.
Successfully installed requests-toolbelt-1.0.0
```

**Comment by mgqa34** [comment]
pip install requests==2.25.1 may  help

**Comment by mgqa34** [comment]
We have released fate-client==1.11.3.post1，thanks  @hyxxsfwy for pointing out this issue.
